merlin/systems/dag/ops/faiss.py

```diff
@@ -131,39 +142,48 @@ def export(
         self_params.update(params)
         index_filename = os.path.basename(os.path.realpath(self.index_path))
 
-        # set index path to new path after export
-        full_path = pathlib.Path(path) / f"{node_id}_{QueryFaiss.__name__.lower()}" / str(version)
+        if backend == "ensemble":
+            full_path = (
+                pathlib.Path(path) / f"{node_id}_{QueryFaiss.__name__.lower()}" / str(version)
+            )
+        else:
+            full_path = pathlib.Path(path) / "executor_model" / str(version) / "ensemble"
 
-        # set index path to new path after export
         new_index_path = full_path / index_filename
-
-        new_index_path.mkdir(parents=True, exist_ok=True)
+        full_path.mkdir(parents=True, exist_ok=True)
         copy2(self.index_path, new_index_path)
         self.index_path = str(new_index_path)
-        return super().export(path, input_schema, output_schema, self_params, node_id, version)
 
-    def transform(self, df: InferenceDataFrame) -> InferenceDataFrame:
+        if backend == "ensemble":
+            return super().export(path, input_schema, output_schema, self_params, node_id, version)
+        else:
+            return ({}, [])
```

Comment:
When we run the FAISS operator in `LocalExecutor` instead of in a Triton ensemble, we still need to execute the `export` method in order to write out the FAISS index file to disk, but we don't need a Triton config for the operator since it no longer has its own directory in the Triton model repository. To make that work with the method signature, we return a null object here.